src/dmd/mars.d

```diff
@@ -1188,7 +1188,14 @@ void addDefaultVersionIdentifiers(const ref Param params)
         VersionCondition.addPredefinedGlobalIdent("Posix");
         VersionCondition.addPredefinedGlobalIdent("linux");
         VersionCondition.addPredefinedGlobalIdent("ELFv1");
-        VersionCondition.addPredefinedGlobalIdent("CRuntime_Glibc");
+        // Note: This should be done with a triplet, to support cross compilation
```

Comment:
What's a "triplet"?

Reply:
https://wiki.osdev.org/Target_Triplet

Reply:
Please put that in the code comment, I doubt I'm the only one wondering what the comment is talking about.